src/services/users.service.js

```diff
@@ -0,0 +1,58 @@
+let users = [];
+let nextId = 1;
+
+const resetUsers = () => {
+  users = [];
+  nextId = 1;
+};
+
+const getUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
+  return users.find((user) => user.id === id);
+};
+
+const createUser = (name) => {
+  const user = { id: nextId++, name };
```

Comment:
Consider adding validation for the `name` parameter to ensure it meets expected formats and constraints (e.g., non-empty, valid characters).